tests/test_ping.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+def test_ping(api):
+    ping = api.resource('ping').call('ping')['results']
+
+    assert ping['foreman']['database']['active']
+
+    if 'katello' in ping:
+      assert ping['katello']['status'] == 'ok'
```

Comment:
How about automatically checking everything?
```suggestion
    for subsystem, data in ping.items():
        try:
            assert data['status'] == 'ok', f"{subsystem} is ok"
        except KeyError:
            # Foreman doesn't have a status - issue #
```

I do think the status labels are localized this may fail if a different locale is used.

And I do think it's ugly that Foreman doesn't have a status. Probably worthy of a bug report. It should combine the database and cache if used (like Redis).